subworkflows/nf-core/abundance_differential_filter/meta.yml

```diff
@@ -12,36 +12,31 @@ components:
   - limma/differential
   - custom/filterdifferentialtable
 input:
-  - ch_abundance:
-      type: file
+  - ch_input:
```

Comment:
I think the new structure for meta.yml for subworkflows is this one: https://github.com/nf-core/modules/pull/6679

Reply:
Oh yep, nice :-)